js/index.js

```diff
@@ -37,6 +37,78 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
+// Images
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let headerImg = document.getElementById("cta-img");
+headerImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let mainImg = document.getElementById("middle-img")
+mainImg.setAttribute('src', siteContent['main-content']['middle-img-src'])
+
+// Nav items
+let navItem = document.querySelectorAll('.container a')
```

Comment:
querySelectorAll was definitely the most efficient way to hit all those anchor tags, great job!